DESCRIPTION

```diff
@@ -27,3 +27,7 @@ Imports:
     jsonlite,
     magrittr,
     testthat
+Suggests: 
```

Comment:
Have you run check() to make sure the package still work?

Reply:
yup! Check still works
<img width="515" alt="Screenshot 2022-01-31 at 17 54 20" src="https://user-images.githubusercontent.com/40150248/151902010-603a9c8e-5a70-42f7-ab71-df03f6f577e1.png">